app/src/components/Results.js

```diff
@@ -1,13 +1,38 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
I would suggest using - import * as React from "react"; instead

Reply:
Thank you very much for your review and suggestions, Alex!